test/end-to-end/test_launcher_checks_supervisor_version.ps1

```diff
@@ -40,6 +40,10 @@ function Test-LauncherFailure($version) {
     return ($supPid.ExitCode -ne 0)
 }
 
+# These tests will timeout unless the launcher and all of its dependencies
+# are already on disk. 
+hab pkg install core/hab-launcher --channel $HAB_BLDR_CHANNEL
+
```

Comment:
This is good but it makes me wonder if we should be adding the sup and launcher to the `setup_environment.ps1` and `.sh` e2e scripts.

Reply:
I thought about that, but thought it best to avoid installing _most_ components in the shared scripts for now in favor of being explicit about what we expect on the system.